app/javascript/packs/player.tsx

```diff
@@ -26,8 +27,9 @@ async function render() {
         portMeta={content.portMeta}
         theme={SEED.theme}
         isOffline={SEED.isOffline}
+        debug={player!.dataset.debug == 'true'}
```

Comment:
what's that `!.` for?

Reply:
Typescript throws a "this could be null" error without it, the `!` overrides the not-null check, telling Typescript to trust that it won't be null